src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,127 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.ValidationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private final String firstValidLogin = "Username";
+    private final String secondValidLogin = "Username2";
+    private final String validPassword = "Password";
+    private final String invalidLogin = "12345";
+    private final String invalidPassword = "12345";
+    private final int validAge = 18;
+    private long id;
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.PEOPLE.clear();
+    }
+
+    @Test
+    void userRegistration_Ok() {
+        User validUser = new User();
+        validUser.setId(id);
+        validUser.setPassword(validPassword);
+        validUser.setLogin(firstValidLogin);
+        validUser.setAge(validAge);
+        registrationService.register(validUser);
+    }
+
+    @Test
+    void sizeStorageWithTwoCorrectUsers_Ok() {
+        User validUser = new User();
+        validUser.setId(id);
+        validUser.setPassword(validPassword);
+        validUser.setLogin(firstValidLogin);
+        validUser.setAge(validAge);
+        User secondValidUser = new User();
+        secondValidUser.setAge(validAge);
+        secondValidUser.setLogin(secondValidLogin);
+        secondValidUser.setPassword(validPassword);
+        secondValidUser.setId(id++);
+        registrationService.register(validUser);
+        registrationService.register(secondValidUser);
+        assertEquals(2, Storage.PEOPLE.size());
+    }
+
+    @Test
+    void getUserByID_Ok() {
+        User validUser = new User();
+        validUser.setId(id);
+        validUser.setPassword(validPassword);
+        validUser.setLogin(firstValidLogin);
+        validUser.setAge(validAge);
+        registrationService.register(validUser);
+        Assertions.assertEquals(validUser,Storage.PEOPLE.get(0));
+    }
+
+    @Test
+    void checkUserIsNull_NotOk() {
+        assertThrows(RuntimeException.class, () -> registrationService.register(null));
+    }
+
+    @Test
+    void checkIsUserLoginLength_notOk() {
```

Comment:
```suggestion
    void register_shortLogin_notOk() {
```
the same with the names of other tests